react/global.css

```diff
@@ -213,3 +213,7 @@ body.vtex-filter-popup-open {
     grid-template-columns: repeat(2, 1fr);
   }
 }
+
+.vtex-dropdown__mobile {
+  margin-top: -2px;
```

Comment:
isit really necessary use a negative margin? I think that could be a problem if the parent component had any change 🤔